src/Twig/Extension/NewsExtension.php

```diff
@@ -19,8 +19,12 @@
 use Sonata\NewsBundle\Model\PostInterface;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\RouterInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\Extension\InitRuntimeInterface;
+use Twig\Environment;
```

Comment:
Please sort use statement alphabetically, this will fix the build. You can do so by running `make cs-fix`